src/checkpoint/checkpoint.h

```diff
@@ -81,6 +81,92 @@ struct __wt_ckpt_connection {
     uint64_t last_base_write_gen;
 };
 
+/*
+ * Block modifications from an incremental identifier going forward.
+ */
```

Comment:
```suggestion
/*
 * WT_CKPT_BLOCK_MODS --
 * Block modifications from an incremental identifier going forward.
 */
```

Reply:
Done

Reply:
The indentation does not look correct. I wonder if it will get caught by `s_all`.
On another note, could you please update the same thing for the `__wt_checkpoint_cleanup` struct as well?

Reply:
s_all only does it for functions.

Reply:
What about:

> On another note, could you please update the same thing for the __wt_checkpoint_cleanup struct as well?

Would you be ok to do it here? Or should we leave it because this is something we will move at some point?

Reply:
All existing comments in `checkpoint.h` are now formatted. `__wt_checkpoint_cleanup` doesn't have one.

I think that "checkpoint cleanup" (or "obsolete cleanup") needs its own refactoring PR where this situation will be clarified and documented.